src/views/Wallet/WalletStats.vue

```diff
@@ -60,7 +59,6 @@ export default {
     ReceiveIcon,
     SwapIcon
   },
-  props: ['loading'],
```

Comment:
I think that we still need the loading indicator but only to show the total fiat balance when all the accounts balances loaded. Tested locally but the total fiat balance seems to be not updated when all balances are loaded

<img width="380" alt="Screen Shot 2021-12-03 at 10 32 11 AM" src="https://user-images.githubusercontent.com/1889891/144629399-7f05ef0c-d2d8-48cf-bc28-d77ad35e6193.png">

Reply:
Hmm that weird, so we need to put back loading indicator for total fiat balance and display only action buttons?
![image](https://user-images.githubusercontent.com/32274987/144816136-5923864f-7461-45c4-b96f-8b0f5f98c58a.png)

Reply:
@bradleySuira can you check if this behaviour is reproducible only for ledger because i don't have one to test that case ;s